src/Compilers/Core/Portable/Syntax/SyntaxNode.cs

```diff
@@ -433,7 +433,7 @@ public bool ContainsDiagnostics
         /// </summary>
         public bool Contains(SyntaxNode node)
         {
-            if (node == null || !this.FullSpan.Contains(node.FullSpan))
+            if (node == null || !this.FullSpan.Contains(node.FullSpan) || node == this)
```

Comment:
> || node == this [](start = 70, length = 16)

It looks like this change broke a lot of existing tests, CI jobs are failing. Also, it looks like @gafter suggested to Won't Fix the issue and update API documentation instead.